src/Page.js

```diff
@@ -148,16 +151,18 @@ function formatSiteNav(renderedSiteNav) {
         // Double wrap to counter replaceWith removing <li>
         nestedList.parent().wrap('<li style="margin-top:10px"></li>');
         // Recursively format nested lists without dropdown wrapper
-        nestedList.parent().replaceWith(formatSiteNav(nestedList.parent().html()));
+        nestedList.parent().replaceWith(formatSiteNav(nestedList.parent().html(), src));
       }
     // Found nested list, render dropdown menu
     } else if ($(this).children('ul').length) {
       const nestedList = $(this).children('ul').first();
       const dropdownTitle = $(this).contents().not('ul');
       // Replace the title with the dropdown wrapper
       dropdownTitle.remove();
-      // Check if dropdown is expanded by default
-      if (dropdownTitle.toString().includes(DROPDOWN_EXPAND_KEYWORD)) {
+      // Check if dropdown is expanded by default or if the current page is in a dropdown
+      const expandDropdown = dropdownTitle.toString().includes(DROPDOWN_EXPAND_KEYWORD)
+        || nestedList.find(`a[href='{{baseUrl}}/${src.replace(/\.(md|mbd)$/, '.html')}']`).text();
```

Comment:
- Wrap in `Boolean(nestedList...text())` to be clear that we're using its truthiness?
- Rename constant to `shouldExpandDropdown` to reflect boolean type.